README.md

```diff
@@ -1,4 +1,6 @@
-<img src="docs/kubescape.png" width="300" alt="logo" align="center">
+<div align="center">
+    <img src="docs/kubescape.png" width="300" alt="logo">
+</div>
```

Comment:
issue: logo is centered, but the badges aren’t

The badges look lonely that way. Let’s add a horizontal rule to visually separate the logo and anchor the badges to the rule.

```suggestion
<div align="center">
    <img src="docs/kubescape.png" width="300" alt="logo">
</div>

---

```